test/test_images/multicontainer/service.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: multi-container-test
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+      - image: ko://knative.dev/serving/test/test_images/multicontainer/servingcontainer
+        ports:
+          - containerPort: 8881
+      - image: ko://knative.dev/serving/test/test_images/multicontainer/sidecarcontainerone
+      - image: ko://knative.dev/serving/test/test_images/multicontainer/sidecarcontainertwo
```

Comment:
Why two sidecars? I'd think one does result in the same test?

Reply:
Yeah I'd drop the second container if it's not interacting with anything